spec/features/shelters/user_can_create_new_review_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'rails_helper'
+
+describe "As a visitor" do
+  describe "When I visit a shelter's show page" do
+    it "I can add a new review for this shelter via link to a form" do
+      shelter = Shelter.create!(
+        name: "Ridiculous Test Name",
+        address: "124 Fake Ln.",
+        city: "Faketown",
+        state: "FK",
+        zip: "55555")
```

Comment:
I'd recommend moving this setup data to a `before(:each) do ... end` block to reduce repetition.

Reply:
Completed.